### DISCUSSION TITLE
Functional Items(Internal tracking)

### DISCUSSION THREAD

**Discussion by mjayakumarmca** [thinkarlabs/P23-003-MalnutritionDashboard#43] [General]
1. Login & Logout functionality for Admin, NGO
2. Admin will be able to see the list of NGO's
3. Admin can add/edit/delete NGO's
4. Admin will be able to see the list of Aganwadi's
5. Admin can add/edit/delete Aganwaidi
6. NGO can login and view Donors/Supplements/Programs
7. NGO can Add/edit/delete Donors/Supplements/Programs
8. User can  login as Aganwadi and see the base page for PWA application

**Comment by srivatsapraveen** [comment]
As discussed last week, I'm still waiting to see the demo script of the aanganwadi mobile app. That takes a higher precedence than the admin features - so can you guys add the aanganwadi features as a higher priority.
